docker/README.md

```diff
@@ -0,0 +1,61 @@
+# Vitess Docker Images
+
+The Vitess Project publishes several Docker images in the [Docker Hub "vitess" repository](https://hub.docker.com/u/vitess/).
+This file describes the purpose of the different images.
+
+**TL;DR:** Use the [vitess/lite](https://hub.docker.com/r/vitess/lite/) image for running Vitess.
+Our Kubernetes Tutorial uses it as well.
+Instead of using the `latest` tag, you can pin it to a known stable version e.g. `v2.0`.
+
+## Principles
+
+The structure of this directory and our Dockerfile files is guided by the following principles:
+
+* The configuration of an image is usually in the directory `docker/<image>/`.
```

Comment:
"usually" means there are exceptions, but there aren't any, right? Or if you meant that the configuration for k8s dependencies and tools are in different directories, then can you rephrase this to tell that? E.g. "The configurations of all Vitess images are in a directory `docker/<image>/` (configurations for Kubernetes dependencies and tools are in other places)."